<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>411</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addTable">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>添加赵云</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletTable">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>220</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>删除赵云</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
